### ISSUE TITLE
The tag below is longer than 255 characters, please use a shorter tag.

### ISSUE THREAD

**Issue by Giraffaman** [Giraffaman/shimmie2#32]
Trying to upload more 80 items, using one common tag:
```
The tag below is longer than 255 characters, please use a shorter tag.
id=678,677,676,675,674,673,672,671,670,669,668,667,666,665,664,663,662,661,660,659,658,657,656,655,654,653,652,651,650,649,648,647,646,645,644,643,642,641,640,639,638,637,636,635,634,633,632,631,630,629,628,627,626,625,624,623,622,621,620,619,618,617,616,615,614,613,612,611,610,609,608,607,606,605,604,603,602,601,600,599
```

**Comment by Giraffaman** [comment]
fixed upstream in 474e3c7

**Comment by Giraffaman** [comment]
popular tags extension seems broken: 
- tag numbers are not refreshed
- new tags are not displayed in the list, but are present in /tags and are searchable
